Content.Server/Clothing/ClothingSolutionTransferSystem.cs

```diff
@@ -0,0 +1,88 @@
+using Content.Server.UserInterface;
+using Content.Shared.FixedPoint;
+using Content.Shared.Interaction;
+using Content.Shared.Popups;
+using Robust.Server.GameObjects;
+using Robust.Shared.GameObjects;
+using Robust.Shared.Localization;
+using Content.Shared.Actions.ActionTypes;
+using Content.Shared.Actions;
+using Content.Shared.Toggleable;
+using Content.Server.Clothing.Components;
+using Content.Shared.Clothing;
+using Content.Shared.Verbs;
+using Content.Server.Nutrition.EntitySystems;
+using Content.Server.Nutrition.Components;
+
+namespace Content.Server.Clothing
+{
+    public sealed class ClothingSolutionTransferSystem : EntitySystem
+    {
+        [Dependency] private readonly DrinkSystem _drinkSystem = default!;
+
+        /// <summary>
+        ///     Default transfer amounts for the set-transfer verb.
+        /// </summary>
+        public static readonly List<int> DefaultTransferAmounts = new() { 1, 5, 10, 25, 50, 100, 250, 500, 1000 };
+
+        public override void Initialize()
+        {
+            base.Initialize();
+
+            SubscribeLocalEvent<ClothingSolutionTransferComponent, GetVerbsEvent<ActivationVerb>>(AddTransferVerb);
+            SubscribeLocalEvent<ClothingSolutionTransferComponent, GetVerbsEvent<AlternativeVerb>>(AddSetTransferVerbs);
+        }
+        //custom verb to move solution from clothing item to wearer
+        private void AddTransferVerb(EntityUid uid, ClothingSolutionTransferComponent component, GetVerbsEvent<ActivationVerb> args)
```

Comment:
Why cant these just use the normal drink verbs? Seems to be the exact same functionality.